Makefile

```
@@ -265,28 +265,32 @@ install-tools:
 generate:
 	go generate ./...
 
-.PHONY: artifacts-darwin
-artifacts-darwin:
-	mkdir -p _artifacts
-	GOOS=darwin GOARCH=amd64 make clean binaries
-	$(TAR) -C _output/ -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-x86_64.tar.gz ./
-	GOOS=darwin GOARCH=arm64 make clean binaries
-	$(TAR) -C _output -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-arm64.tar.gz ./
+.PHONY: artifacts-darwin artifact-darwin-arm64 artifact-darwin-x86_64
+artifacts-darwin: artifact-darwin-x86_64 artifact-darwin-arm64
```

I know this is a trivial thing, but it bothers me that the names are listed in different orders in these 2 lines:

```suggestion
.PHONY: artifacts-darwin artifact-darwin-arm64 artifact-darwin-x86_64
artifacts-darwin: artifact-darwin-arm64 artifact-darwin-x86_64
```

Same thing for the Linux artifacts.

---

Changing that would alter the build order compared to before.
Additionally, personally, since I use the path to `_output/bin` in the working directory, I prefer having `arm64` built later. I don't need to rebuild after running `make artifacts-darwin`.😃

---

You could also change the order in the `.PHONY` directive to match the build order. 😄 

But it isn't that important...